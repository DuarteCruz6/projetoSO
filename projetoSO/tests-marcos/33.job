WRITE [(a9,72)(l5,29)(u4,63)(e9,74)(g3,100)]
WRITE [(z5,1)(x1,63)(e5,93)(i7,25)(m8,58)]
WRITE [(m9,73)(e6,15)(u9,26)(t0,80)(v2,70)]
WRITE [(s5,3)(p2,1)(y3,8)(w2,79)]
WAIT 3000
WRITE [(q0,51)(g7,85)(w7,91)(b5,95)]
WRITE [(x9,53)(b8,89)(u0,4)(l4,0)(q3,40)]
READ [q0]
DELETE [i7,q3]
WRITE [(z7,2)(i1,90)(m2,36)(f2,46)(l8,50)]
