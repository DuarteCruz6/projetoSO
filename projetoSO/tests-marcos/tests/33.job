WRITE [(h5,79)(w7,63)(c8,90)(y6,84)]
SHOW
WRITE [(l5,61)(u0,74)]
DELETE [l5,h5,u0]
WRITE [(f5,56)(f6,62)(a4,15)]
READ [c8]
WRITE [(o3,80)(c2,19)(d7,43)]
WRITE [(k6,78)(w2,76)(a8,53)]
WRITE [(d4,75)(y4,65)(j4,11)(n8,47)(q4,85)]
READ [f5]
DELETE [o3,a8,d7,w7]
READ [q4]
DELETE [c8,q4,j4,k6,d4]
WRITE [(k5,99)(g9,46)(m2,65)(g4,64)]
WRITE [(l1,38)(z3,33)(q7,50)(j9,6)]
DELETE [q7,g9,l1,z3,y4]
DELETE [f6,c2,j9,y6,k5]
WRITE [(b0,70)(c9,71)]
WRITE [(u2,41)]
DELETE [u2]
WRITE [(s3,40)(y9,4)(y5,9)]
DELETE [f5,c9,g4,y9]
SHOW
SHOW
WRITE [(h8,86)(z2,99)(p2,82)(x0,19)]
WRITE [(z1,0)]
WRITE [(x8,23)(j2,22)(y7,30)]
WRITE [(t6,60)(q4,87)(x4,63)(v4,29)(r1,84)]
WRITE [(x2,43)]
BACKUP
WRITE [(r5,56)(h7,27)(l4,46)(f0,83)]
READ [a4]
WRITE [(w8,5)(a1,16)(e1,36)]
READ [w8]
WRITE [(c4,22)(k8,16)(l3,67)]
DELETE [z1,x2,w8,x8,p2]
BACKUP
WRITE [(n4,29)]
WRITE [(x9,100)(g0,83)(o2,47)(l8,16)]
WAIT 2000
WRITE [(i3,82)(l1,46)]
WRITE [(j6,43)(s4,77)(b5,52)(x2,99)(m3,58)]
DELETE [l1,v4]
