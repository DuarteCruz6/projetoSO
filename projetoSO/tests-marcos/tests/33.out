(c8, 90)
(h5, 79)
(w7, 63)
(y6, 84)
[(c8,90)]
[(f5,56)]
[(q4,85)]
(a4, 15)
(b0, 70)
(m2, 65)
(n8, 47)
(s3, 40)
(w2, 76)
(y5, 9)
(a4, 15)
(b0, 70)
(m2, 65)
(n8, 47)
(s3, 40)
(w2, 76)
(y5, 9)
[(a4,15)]
[(w8,5)]
